src/main/resources/db/migration/NTV20231120_0001__GA-28_replacement_index_for_case_access_groups.sql

```diff
@@ -0,0 +1,2 @@
+DROP INDEX IF EXISTS idx_case_data_case_access_groups;
```

Comment:
This file needs to be renamed to match the naming convention used in the project.